src/containers/Search.jsx

```diff
@@ -228,7 +241,8 @@ class Search extends Component {
 		} = this.state;
 		const result_list =
 			results && results.length !== 0
-				? results.map(
+				? results.sort((a,b)=>a.dist-b.dist)
```

Comment:
maybe use absolute value;

Reply:
it is sorted based on if it is a negative or positive number, if I use abs it will always put 'a' at the front

Reply:
the way you do sorting here is to find relevant distances from user location, find a diff/ in distance from user location. it can never be < zero.